packages/appium/docs/scripts/build-reference.js

```diff
@@ -0,0 +1,33 @@
+const {fs} = require('@appium/support');
+const {log, LANGS, DOCS_DIR} = require('./utils');
+const path = require('path');
+const monorepoRoot = path.resolve(__dirname, '..', '..', '..', '..');
+const typedocOut = path.resolve(monorepoRoot, 'typedoc-docs');
+const {exec} = require('teen_process');
+
+const typedocMap = [[['commands', 'appium_base_driver.md'], ['base_driver.md']]];
```

Comment:
I think this is OK for now, but also we can likely implement more granular control over file names so this is not needed.

...but I'm also unsure why we wouldn't just configure typedoc to put the file where we want it?

Reply:
if we can do that, that's preferable! seemed like there was only one `out` option for typedoc, and we are pulling stuff from multiple packages into that for convenience. but if we can put e.g. the fake_driver.md into the fakedriver docs, and base_driver.md into the appium docs, then we should just do that, which would simplify this script a whole lot